app/test_main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+import pytest
```

Comment:
Imports must be separated + There must be blank lines between different import types